cmd/kube-apiserver/app/aggregator.go

```diff
@@ -67,6 +67,9 @@ func createAggregatorConfig(
 	genericConfig := kubeAPIServerConfig
 	genericConfig.PostStartHooks = map[string]genericapiserver.PostStartHookConfigEntry{}
 	genericConfig.RESTOptionsGetter = nil
+	// prevent generic API server from installing the OpenAPI handler. Aggregator server
+	// has its own customized OpenAPI handler.
+	genericConfig.SkipOpenAPIInstallation = true
```

Comment:
Re:  second sentence of [your suggestion](https://github.com/kevindelgado/kubernetes/commit/0e0248817b9f396b041295bc78486908e426e1d0#r48352264) 

In the specific case of the aggregator, I assume we want to set it in `createAggregatorConfig` rather than where we set `k8s.io/apiserver/pkg/server#Config.OpenAPIConfig` in [buildGenericConfig](https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-apiserver/app/server.go#L474) because that config is used by all the apiservers.

Reply:
I was envisioning where kube-aggregator sets `k8s.io/apiserver/pkg/server#Config.OpenAPIConfig`:

https://github.com/kubernetes/kubernetes/blob/2f8a2258f45b409481cd66be7d34a5b482db3899/staging/src/k8s.io/kube-aggregator/pkg/cmd/server/start.go#L143-L144

Reply:
oh interesting, done.

Just for my own curiosity, does kube-apiserver call `RunAggregator()` anywhere? I don't see it called and it's unclear how kube-apiserver makes use of config setting done in `RunAggregator()`